igneous/tasks/skeleton.py

```diff
@@ -832,10 +1091,10 @@ def execute(self):
     skeletons = []
     for segid, frags in skels.items():
       skeleton = self.fuse_skeletons(frags)
-      if self.max_cable_length is None or skeleton.cable_length() <= self.max_cable_length:
-        skeleton = kimimaro.postprocess(
-          skeleton, self.dust_threshold, self.tick_threshold
-        )
+      # if self.max_cable_length is None or skeleton.cable_length() <= self.max_cable_length:
```

Comment:
Maybe add a parameter for postprocessing type or "skip_postprocess"? I think the main problem here was loop elimination is O(V^2) 

Same comment as in sharded.